components/brave_wallet/browser/eth_json_rpc_controller.h

```diff
@@ -27,7 +28,7 @@ class SimpleURLLoader;
 
 namespace brave_wallet {
 
-class EthJsonRpcController {
+class EthJsonRpcController : public url_loader_component::URLLoaderComponent {
```

Comment:
Not sure it makes sense to use inheritance instead of just adding this object as a field (aka composition)